### ISSUE TITLE
"@nativescript/types": "8.3.0" broken

### ISSUE THREAD

**Issue by cjohn001** [NativeScript/NativeScript#9974]
### Issue Description

I just tried to upgrade a plugin to 

"@nativescript/types": "8.3.0"

I receive the following error message when building for ios:

```
node_modules/@nativescript/types-ios/lib/ios/objc-x86_64/objc!Darwin.d.ts:4410:2 - error TS1131: Property or signature expected.
4410  : number;
      ~
node_modules/@nativescript/types-ios/lib/ios/objc-x86_64/objc!Darwin.d.ts:4413:2 - error TS1131: Property or signature expected.
4413  : number;
      ~
node_modules/@nativescript/types-ios/lib/ios/objc-x86_64/objc!Darwin.d.ts:4414:2 - error TS1131: Property or signature expected.
4414  : number;
      ~
Found 3 errors in the same file, starting at: node_modules/@nativescript/types-ios/lib/ios/objc-x86_64/objc!Darwin.d.ts:4410
```
When looking into the relevant file one finds the following interface definition.

```
interface __darwin_fp_control {
	__invalid: number;
	__denorm: number;
	__zdiv: number;
	__ovrfl: number;
	__undfl: number;
	__precis: number;
	: number;
	__pc: number;
	__rc: number;
	: number;
	: number;
}
```

Hence, it seems like there is something broken with the type generator. There are three unnamed number values defined in the interface.

### Reproduction

Install package and build with an app. I use

`/// <reference path="./node_modules/@nativescript/types-ios/complete.d.ts" />
`

### Environment

```yaml
OS: macOS 12.5
CPU: (10) arm64 Apple M1 Pro
Shell: /bin/zsh
node: 18.1.0
npm: 8.8.0
nativescript: 8.3.0

# android
java: 11.0.15
ndk: Not Found
apis: Not Found
build_tools: Not Found
system_images: Not Found

# ios
xcode: 13.4.1/13F100
cocoapods: 1.11.3
python: 3.9.12
python3: 3.9.12
ruby: 2.7.6
platforms: 
  - DriverKit 21.4
  - iOS 15.5
  - macOS 12.3
  - tvOS 15.4
  - watchOS 8.5
```

### Dependencies

```json
"dependencies": {
  "md5": "^2.3.0"
},
"devDependencies": {
  "@nativescript/core": "~8.3.1",
  "@nativescript/types": "~8.3.0",
  "typescript": "4.6.4",
  "rimraf": "^3.0.2",
  "tslint": "^6.1.3",
  "@types/md5": "^2.3.2"
}
```

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by NathanWalker** [comment]
Thanks for reporting @cjohn001 publishing a patch to resolve this shortly 👍

**Comment by NathanWalker** [comment]
published in @nativescript/types ~8.3.1

**Comment by cjohn001** [comment]
Thanks for the quick update!
